Procurement/ViewModel/Recipes/VaalOrbRecipe.cs

```diff
@@ -57,7 +55,7 @@ public override IEnumerable<RecipeResult> Matches(IEnumerable<Item> items)
                     recipeResult.Missing.Add("Sacrifice Fragment");
                 }
 
-                recipeResult.PercentMatch = (decimal) ((recipeResult.MatchedItems.Count / 8d) * 100) ;
+                recipeResult.PercentMatch = (decimal) (recipeResult.MatchedItems.Count / 8d * 100) ;
```

Comment:
`NeededVaalGems + 1d` instead of `8d` (or even also add a member for the number of needed sacrifice fragments).